foo_ui_columns/drop_down_list_toolbar.h

```diff
@@ -17,6 +17,22 @@ class DropDownListToolbar : public ui_extension::container_ui_extension {
         }
     }
 
+    static void s_update_colours()
+    {
+        if (s_background_brush != nullptr)
+            DeleteObject(s_background_brush);
```

Comment:
If you fancy it you could use `wil::unique_hbrush` for `s_background_brush`.

Assigning a new HBRUSH to it would become:

```cpp
s_background_brush.reset(CreateSolidBrush(helper.get_colour(cui::colours::colour_background)));
```

([It's documented here.](https://github.com/microsoft/wil/wiki/RAII-resource-wrappers#usage-1))

Reply:
A RAII wrapper? This should simplify things. :+1: